partcad/src/partcad/cache_hash.py

```
@@ -13,18 +13,37 @@
         self.is_empty = True
         self.is_used = False
+        # Set before the early return below: get() walks this list, and it is
+        # reached with caching disabled too (a disabled hash still answers
+        # None, it just never hashes anything).
+        self.dependencies = []
         if not cache:
             # Caching is disabled, no initialization needed
             self.hasher = None
             return
 
         if hasher != None:
```

Ruff (0.15.21)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)